build.gradle

```diff
@@ -1,3 +1,31 @@
+// make folders
+
+task mkdir {
+    group = "kami"
+    doLast {
+        mkdir "../../.minecraft/mods/1.12.2"
+    }
+}
+
+// delete old versions
+
+task rmold(type: Delete) {
+    group = "kami"
+    delete fileTree("../../.minecraft/mods/1.12.2").matching {
+        include "KAMI*.jar"
+    }
+}
+
+// copy file
+
+task copy(type: Copy) {
+    group = "kami"
+    from file("build/libs/KAMI-b9-release.jar")
+    into file("../../.minecraft/mods/1.12.2/")
```

Comment:
We can't just assume `../../.minecraft` is the correct path to our minecraft directory.
Use `System.getProperty("user.home") + "/.minecraft"` instead

Reply:
I'm not sure how reviews work on github, do I edit it myself?

Reply:
Just commit and mark the conversation as resolved.